### PR TITLE
docs: first look at remap.adoc

### PR DESCRIPTION
**Pull request by smoe**
Needs to be merged prior to continue investing into weblate.

### CODE REVIEW

File: docs/src/remap/remap.adoc [hunk 1]
```diff
@@ -2678,15 +2674,15 @@ thing.
...
+RANDOM_TOOLCHANGER INI option was set, but at the end of the process
 'toolTable[0]' reflects the tool currently in the spindle.
 
 When that has happened:
 
-. iocontrol signals task to go ahead
-. task tells the interpreter to execute a 'synch()' operation, to see
-  what has changed
-. the interpreter 'synch()' pulls all information from the world model
+. Iocontrol signals task to go ahead.
```
Comment (hansu) [inline docs/src/remap/remap.adoc:2682]: I/O control

Comment (smoe) [reply docs/src/remap/remap.adoc:2682]: I think this refers to an iocontrol HAL module. Maybe `iocontrol` and I should just not capialize despite the colon?

Comment (smoe) [reply docs/src/remap/remap.adoc:2682]: same for `task`.

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> I think this refers to an iocontrol HAL module. Maybe `iocontrol` and
> I should just not capialize despite the colon?

What about being explicit, in other words write 'The iocontrol component
signals task to go ahead'?  Then we do not have to guess if it refer to
the iocontrol component.

-- 
Happy hacking
Petter Reinholdtsen

File: docs/src/remap/remap.adoc (hunk 1 above)
Comment (hansu) [reply docs/src/remap/remap.adoc:2682]: Sure, I only had the title of https://linuxcnc.org/docs/html/config/iov2.html in mind which says "I/O control".
But yeah, just a suggestion...

**Comment by hansu** [comment]
>What about being explicit, in other words write 'The iocontrol component
signals task to go ahead'? 

:+1: 
Or format it italic oder as code

**Comment by c-morley** [comment]
Why would you remove the information about synch()?
 Is is referring to the interpeter's synch() method (earlier in document:)

The interpreter’s synch() method is called by task and does just that - read all the world model actual values which are relevant for further execution.

When synch() is run, I dare says is an important detail.

**Comment by smoe** [comment]
> Why would you remove the information about synch()? Is is referring to the interpeter's synch() method (earlier in document:)
> 
> The interpreter’s synch() method is called by task and does just that - read all the world model actual values which are relevant for further execution.
> 
> When synch() is run, I dare says is an important detail.

Thank you for spotting that. Synch is mentioned twice in the paragraph, the first time without the ()s, and I did not get that this was important when the autotranslators have just completed it to synchronization which I then also clarified for the English. So - many thanks again. Well done.

I'll do some more reading so that I understand this myself and then come up with something that is autotranslator-proof. ;) Please feel free to beat me to it.

**Comment by c-morley** [comment]
ok, ya that totally fooled me :)
Sorry for the noise.

File: docs/src/remap/remap.adoc [hunk 2]
```diff
@@ -2343,15 +2343,15 @@ FIXME Write missing information
 
 FIXME Write missing information
 
-=== A diagram: task, interp, iocontrol, UI (??)
+=== A diagram: `task`, `interp`, `iocontrol`, `UI` (??)
```
Comment (hansu) [inline docs/src/remap/remap.adoc:2346]: Not sure if we should have monospaced font in headings. Maybe something for the style guide ;-)

**Comment by silopolis** [comment]
Le dim. 4 sept. 2022 à 21:17, Hans Unzner ***@***.***> a
écrit :

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In docs/src/remap/remap.adoc
> <https://github.com/LinuxCNC/linuxcnc/pull/1990#discussion_r962357557>:
>
> > @@ -2343,15 +2343,15 @@ FIXME Write missing information
>
>  FIXME Write missing information
>
> -=== A diagram: task, interp, iocontrol, UI (??)
> +=== A diagram: `task`, `interp`, `iocontrol`, `UI` (??)
>
> Not sure if we should have monospaced font in headings. Maybe something
> for the style guide ;-)
>
I believe we do and quite a bunch are coming...

Maybe we could take advantage of GH features and tag messages related to
style guide with PR id #1530 ?!

File: docs/src/remap/remap.adoc (hunk 2 above)
Comment (hansu) [reply docs/src/remap/remap.adoc:2346]: #1530

**Pull request merged**